README.md

```diff
@@ -65,21 +65,37 @@ Load Marduk platform specific OpenWrt configuration for Pistachio.
         Target Profile (Basic platform profile for Marduk)  ---> 
             (X) Basic platform profile for Marduk with TI cc2520
             ( ) Basic platform profile for Marduk with Cascoda ca8210
+            ( ) Wifi testing profile for Marduk with TI cc2520
 
-You can also load Marduk platform specific OpenWrt configuration for Pistachio by adding following into .config file.
+You can also load Marduk platform (with TI cc2520) specific OpenWrt configuration for Pistachio by adding following into .config file.
 
     $ echo "CONFIG_TARGET_pistachio=y" > .config
     $ echo "CONFIG_TARGET_pistachio_marduk_marduk_cc2520=y" >> .config
 
+You can also load Marduk platform (with Cascoda ca8210) specific OpenWrt configuration for Pistachio by adding following into .config file.
```

Comment:
Update this line "Similarly, you can do the same for Marduk platform with Cascoda ca8210 specific OpenWrt configuration:"